.eleventy.js

```diff
@@ -1,38 +1,53 @@
-const { isDev } = require('./pages/views/_data/env')
+const { isDev } = require("./pages/views/_data/env");
+const htmlmin = require("html-minifier");
 
-module.exports = config => {
+module.exports = (config) => {
   config.addPassthroughCopy({
-    'pages/static/assets': 'assets',
-    'pages/static/tools': '.',
+    "pages/static/assets": "assets",
```

Comment:
Is it compliant with our editor config rules?
If I'm not mistaken we should use single quotes for JS and TS

Reply:
Thank you for good note. It's my mistake and it will be fixed.